osu.Game/Screens/OnlinePlay/Lounge/Components/RoomPanel.cs

```diff
@@ -384,17 +383,17 @@ public virtual MenuItem[] ContextMenuItems
                 if (Room.RoomID.HasValue)
                 {
                     items.AddRange([
-                        new OsuMenuItem("View in browser", MenuItemType.Standard, () => game?.OpenUrlExternally(formatRoomUrl(Room.RoomID.Value))),
-                        new OsuMenuItem("Copy link", MenuItemType.Standard, () => game?.CopyToClipboard(formatRoomUrl(Room.RoomID.Value)))
+                        new OsuMenuItem("View in browser", MenuItemType.Standard, () => game?.OpenUrlExternally(getRoomUrl())),
+                        new OsuMenuItem("Copy link", MenuItemType.Standard, () => game?.CopyToClipboard(getRoomUrl()))
                     ]);
                 }
 
                 return items.ToArray();
-
-                string formatRoomUrl(long id) => $@"{api.Endpoints.WebsiteUrl}/multiplayer/rooms/{id}";
             }
         }
 
+        private string? getRoomUrl() => !Room.RoomID.HasValue ? null : $@"{api.Endpoints.WebsiteUrl}/multiplayer/rooms/{Room.RoomID.Value}";
```

Comment:
Make this non-nullable, static, and have an input variable with the `Room`. The nullable return here is really weird (none of the usages can/should ever be null).

Reply:
Hmm good catch, might be best to move this to an extension similar to `BeatmapInfoExtensions.GetOnlineURL`.